.github/workflows/build.yml

```diff
@@ -48,15 +48,19 @@ jobs:
         run: |
           if [ "$GITHUB_REF_TYPE" = "tag" ]; then
             arrTag=(${GITHUB_REF//\// })
-            VERSION="${arrTag[2]}"
-            echo Version: $VERSION
-            VERSION="${VERSION//v}"
-            echo Clean Version: $VERSION
+            version="${arrTag[2]}"
+            echo Version: $version
+            version="${version//v}"
+            echo Clean Version: $version
           else
-            VERSION=$GITHUB_RUN_ID
-            echo Non-release version: $VERSION
+            git fetch --prune --unshallow --tags --quiet
+            latestTag=$(if [ "$(git tag -l | wc -l)" -gt 0 ]; then git describe --tags --abbrev=0; else echo 0.0.1; fi)
```

Comment:
This will fail if there are tags (=> `git tag -l` succeeds) but none of them are a parent of the current commit (=> `git describe` fails).

This should work better: `git describe --tags --abbrev=0 2>/dev/null || echo 0.0.1`

Reply:
Oh boy, I think you are right on this one!